### PR TITLE
feat(replays): Adds some spans as breadcrumbs

### LINKED ISSUE #35511: Add LCP/FCP into breadcrumbs
I think we'll want to do this, but let's wait until SDK gets refactored with non-transaction method. These currently exist in spans, but we should also create breadcrumb entries for them. We can then also highlight the LCP element instead of including it in the playback which was always a bit spotty.

### LINKED ISSUE #35681: Timeline tooltip is empty for the `ui.exit` click event

### PR DESCRIPTION
**Pull request by billyvg**
Creates breadcrumbs from specific spans (e.g. LCP and navigation). Refactors components to support this + some cleanups as well.

image

Closes https://github.com/getsentry/sentry/issues/35511
Closes https://github.com/getsentry/sentry/issues/35681

### CODE REVIEW

File: static/app/utils/replays/replayDataUtils.tsx [hunk 1]
```diff
@@ -53,39 +43,87 @@ export function breadcrumbFactory(
...
     timestamp: new Date(startTimestamp).toISOString(),
     level: BreadcrumbLevelType.INFO,
-    action: 'replay-init',
-    message: t('Start recording'),
+    message: initialUrl,
     data: {
-      url: tags.find(tag => tag.key === 'url')?.value,
+      action: 'replay-init',
+      label: t('Start recording'),
+      url: initialUrl,
```
[the remark this answers is not in the document]
Comment (ryan953) [reply static/app/utils/replays/replayDataUtils.tsx:59]: the timestamp in here is the interesting part to keep, as well as making the guarantee that there will always be one breadcrumb.

i'd want to bounce back to the browser api's that the sdk calls to make sure we're guaranteed to get lcp, or navigate events to matter how the sdk is used.

[the remark this answers is not in the document]
Comment (billyvg) [reply static/app/utils/replays/replayDataUtils.tsx:59]: I mean the url property specifically

File: static/app/views/replays/detail/breadcrumbs/index.tsx [hunk 2]
```diff
@@ -65,27 +50,28 @@ function UserActionsNavigator({event, crumbs}: Props) {
   } = useReplayContext();
 
   const startTimestamp = event?.startTimestamp || 0;
-  const userActionCrumbs =
-    crumbs?.filter(crumb => USER_ACTIONS.includes(crumb.type)) || [];
 
   const isLoaded = Boolean(event);
 
+  const crumbs =
+    allCrumbs?.filter(crumb => !['console'].includes(crumb.category || '')) || [];
```
[the remark this answers is not in the document]
Comment (ryan953) [reply static/app/views/replays/detail/breadcrumbs/index.tsx:57]: this'll also fix that [Render custom breadcrumbs that the SDK captured #35746](https://github.com/getsentry/sentry/issues/35746)

**Review by ryan953 — APPROVED** [review]
:shipit: :shipit: :shipit: 🚀

File: static/app/views/replays/detail/breadcrumbs/index.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (billyvg) [reply static/app/views/replays/detail/breadcrumbs/index.tsx:57]: yep! Although there may be some surprises that we'll have to follow-up on.

**Pull request merged**